packages/react-universal-provider/src/create-universal-provider.tsx

```diff
@@ -8,13 +8,13 @@ export interface UniversalProviderProps<Value> {
 
 export function createUniversalProvider<Value>(
   id: string,
-  Context: React.Context<Value | null>,
+  Context: React.Context<Value | undefined>,
 ) {
   const UniversalProvider = React.memo(
     ({value: explicitValue, children}: UniversalProviderProps<Value>) => {
       const [value = explicitValue, Serialize] = useSerialized<Value>(id);
 
-      if (!value) {
+      if (value === undefined) {
```

Comment:
I been wonder if this is valuable to have at all. If a consumer use `UniversalProvider` with null value, would that ever be on purpose?

Reply:
I switched this to `value == null` instead. I was concerned `value` could be a `boolean`.